src/main/java/dk/kb/util/webservice/Service2ServiceRequest.java

```diff
@@ -44,24 +44,30 @@ public class Service2ServiceRequest {
     * @param uri the full URI with path and parameters set.
     * @param httpMethod The http-method to use for the service call. GET, POST, DELETE etc.
     * @param objectClass The DTO type that the response should be parsed to.
-    * @return DtoObject (objectClass) of the same type at given as input. 
+    * @param postJsonDto If not the DTO will be send as POST. Set to null if API method does not require a DTO. 
```

Comment:
I dont quite understand what you mean when you start the sentence with "if not the DTO will de send ..." My question here is: If not what?

Reply:
reformulated.